iothub/device/src/Common/Api/ClientApiVersionHelper.cs

```diff
@@ -7,8 +7,11 @@ internal class ClientApiVersionHelper
     {
         internal const string ApiVersionQueryPrefix = "api-version=";
         internal const string ApiVersionLatest = "2019-10-01";
+        internal const string ApiVersionPnP = "2020-05-31-preview";
 
         public const string ApiVersionString = ApiVersionLatest;
-        public const string ApiVersionQueryString = ApiVersionQueryPrefix + ApiVersionString;
+        public const string ApiVersionQueryStringLatest = ApiVersionQueryPrefix + ApiVersionString;
+
+        public const string ApiVersionQueryStringPnP = ApiVersionQueryPrefix + ApiVersionPnP;
```

Comment:
Maybe the postfix here should be "Preview" instead of "PnP"

Reply:
I did consider this, but it isn't really a preview API version, as in, we don't claim support for "preview" features such as device streaming etc. So, adding "pnp" as postfix made more sense to me.